munit/shared/src/main/scala-3/munit/internal/MacroCompat.scala

```diff
@@ -1,11 +1,17 @@
 package munit.internal
 
 import munit.Clue
+import munit.Compare
 import munit.Location
 import scala.quoted._
 
 object MacroCompat {
 
+  trait CompareMacro {
+    implicit def defaultCompareImplicit[A, B](using eql: Eql[A, B]): Compare[A, B] =
+      Compare.defaultCompare[A, B]
+  }
```

Comment:
Not it's idiomatic, but I think in Scala 3 we can do:

```suggestion
  trait CompareMacro {
    given [A, B](using eql: Eql[A, B]) as Compare[A, B] = Compare.defaultCompare[A, B]
  }
```

Reply:
TIL, I updated the implementation accordingly!